core/sql/optimizer/OptimizerSimulator.cpp

```diff
@@ -278,9 +277,10 @@ OptimizerSimulator::OptimizerSimulator(CollHeap *heap)
     sysCallLogFilePath_[sc]=NULL;
     writeSysCallStream_[sc]=NULL;
   }
+  // Init log4cxx logging
+  CommonLogger::instance().initLog4cxx("log4cxx.trafodion.osim.config");
```

Comment:
Isn't this running in the Trafodion process, master executor or compiler? If so, won't that interfere with the logger already established for the process? I'm wondering whether we need a separate logger for OSIM or whether we could just reuse the existing one?